docs/src/_includes/components/rule-categories.macro.html

```diff
@@ -32,7 +32,8 @@
     <div class="rule-category">
         <span class="rule-category__icon">❄️ <span class="visually-hidden">Frozen</span></span>
         <p class="rule-category__description">
-            This rule is currently frozen and is not accepting changes.
+            This rule is currently frozen and is not accepting <a
+                href="https://github.com/eslint/eslint/#frozen-rule-updates">feature requests</a>
```

Comment:
> https://github.com/eslint/eslint/#frozen-rule-updates

[Suggestion] I do find it a little odd to have the _eslint.org_ website link to the GitHub repository. That's not common on rule pages, and certainly not this high up in the page. How about instead linking to a good resource on the website, maybe <https://eslint.org/docs/latest/use/core-concepts/glossary#stylistic-rule>?